src/main/scala/tech/cryptonomic/conseil/michelson/JsonToMichelson.scala

```diff
@@ -0,0 +1,15 @@
+package tech.cryptonomic.conseil.michelson
+
+import tech.cryptonomic.conseil.michelson.parser.JsonParser
+import tech.cryptonomic.conseil.michelson.presenter.MichelsonPresenter._
+
+class JsonToMichelson {
+
+  private val parser = new JsonParser()
+
+  type Result[T] = Either[Throwable, T]
+
+  def convert(json: String): Result[String] = parser
```

Comment:
In a subsequent PR, could we actually invoke this logic when we save scripts and params into the `accounts` and `operations` tables?